mRemoteV1/UI/Window/ConnectionWindow.cs

```diff
@@ -419,11 +421,11 @@ private void ShowHideMenuButtons()
 
                 if (interfaceControl.Protocol is PuttyBase)
                 {
-                    cmenTabPuttySettings.Visible = true;
+                    cmenCopyAllToClipboard.Visible = cmenTabPuttySettings.Visible = true;
```

Comment:
These options should be in addition to cmenTabPuttySettings.Visible not in place of them...

Reply:
Sorry, I don't really get it. Would you prefer this dissected in two lines?

Reply:
I did say "a super quick code review" in my other comment... :wink:

I missed the inline setter calls for both components. All good here...